src/lib/User/model.ts

```diff
@@ -15,6 +15,7 @@ export interface User extends Document {
   name: string;
   email: string;
   password: string;
+  salt: string;
```

Comment:
Do we need to store the salt?

Reply:
His method was generating a salt each time, so I think we do.

Reply:
Storing the salt alongside the hashed password simplifies the verification process during login attempts.